package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@crowdstrike/glide-core",
   "version": "0.19.5",
-  "description": "CrowdStrike's Glide Design System components package for providing web components",
+  "description": "A Web Component design system",
```

Comment:
Simplified it a bit for NPM's typeahead. I figured that it's from CrowdStrike is evident from the package name. Yay? Nay? 

<img width="920" alt="image" src="https://github.com/user-attachments/assets/26ed3f96-6d45-4112-bb0b-e6cada1deaf5" />